crates/oxc_language_server/src/main.rs

```diff
@@ -67,11 +67,42 @@ impl Options {
         }
     }
 
-    fn get_config_path(&self) -> Option<PathBuf> {
-        if self.config_path.is_empty() {
+    fn get_config_path(&self, root_path: &Path) -> Option<PathBuf> {
+        let Some(config_path) = &self.config_path else {
+            // no config file is provided, let search in the root_path for one
+            let search_configs = vec![
+                "oxlintrc.json",
+                "oxlint.json",
+                ".oxlintrc.json",
+                ".oxlint.json",
+                ".oxlintrc",
+                ".eslintrc",
+            ];
+
+            for search_config in search_configs {
+                let config_path = root_path.join(search_config);
+
+                if config_path.exists() {
```

Comment:
This returns the first config file that exists, but there might be more than one match.  Should that be communicated somehow?

`Using oxlintrc.json, but found oxlint.json too. Specify oxlint.json in your settings to use it` or something like that as a log message.  It has some additional overhead, but could be useful for users.

I think just logging which config is being used could be helpful too. `Using oxlintrc.json` or `Automatically detected oxlintrc.json`.

Reply:
added `info!("Automatically detected config {config_path:?}");`